subprojects/common/analysis/src/main/kotlin/hu/bme/mit/theta/analysis/multi/builder/MultiBuilderResult.kt

```diff
@@ -0,0 +1,89 @@
+/*
+ *  Copyright 2024 Budapest University of Technology and Economics
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package hu.bme.mit.theta.analysis.multi.builder
+
+import hu.bme.mit.theta.analysis.Action
+import hu.bme.mit.theta.analysis.Prec
+import hu.bme.mit.theta.analysis.State
+import hu.bme.mit.theta.analysis.multi.*
+import hu.bme.mit.theta.analysis.multi.config.MultiBlankInitFunc
+import hu.bme.mit.theta.analysis.unit.UnitPrec
+import hu.bme.mit.theta.analysis.unit.UnitState
+
+// The commented class in this file would be the appropriate solution, however, with
+//IntelliJ IDEA 2024.1 (Ultimate Edition)
+//Build #IU-241.14494.240, built on March 28, 2024
+//Runtime version: 17.0.10+8-b1207.12 amd64
+//VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
+//Windows 10.0
+//GC: G1 Young Generation, G1 Old Generation
+//Memory: 12288M
+//Cores: 12
+//Registry:
+//debugger.new.tool.window.layout=true
+//ide.experimental.ui=true
+//Non-Bundled Plugins:
+//com.github.lonre.gruvbox-intellij-theme (0.6.2)
+//Kotlin: 241.14494.240-IJ
+// the usage of this class, or any other solution in this file causes the kotlin plugin to crash, maybe even crashing
+// the whole IDE. For this reason, a POJO has to be used instead
+
+//data class MultiBuilderResult
```

Comment:
remove this commented class

Reply:
Made statement more clear in 5be9f68af8765970cb72f7c708f14da31dfa632e

Reply:
Resolved in ed176c32f0cf7cf68ac3cbc1dacb512af1ba9d98